components/scrapd-home-view/component.js

```diff
@@ -29,41 +34,40 @@ const ScrapdHomeView = () => {
   const grabData = [
     <div key="0">
       <p>
-        We are fully transparent with our data sets. We publish them on{' '}
+        The scrAPD web scraper generates data sets which populate our scrAPD dashboard. Use the “JSON” and “CSV” buttons
```

Comment:
We need to be consistent and write `ScrAPD` with the same casing everywhere.

Reply:
Complete